submissions/Zakkarat/OOP-exercise/index.js

```diff
@@ -45,25 +66,12 @@ const Character = function (name, gender, legs, saying) {
     }
```

Comment:
I'm not sure that is good idea to mess prototype style with es6 classes. I think it would be better, put you sentence to class character in constructor or method of class.